rest-api-spec/src/main/resources/rest-api-spec/test/indices.delete/10_basic.yml

```diff
@@ -0,0 +1,135 @@
+setup:
+  - do:
+      indices.create:
+        index: index
+        body:
+          aliases:
+            alias: {}
+  - do:
+      indices.create:
+        index: index2
+---
+"Delete index against alias":
+  - skip:
+      version: " - 5.99.0"
+      reason: delete index doesn't support aliases only from 6.0.0 on
+  - do:
+      catch: missing
+      indices.delete:
+        index: alias
+  - do:
+      indices.get:
+        index: index,index2
+  - is_true: index
+  - is_true: index2
+---
+"Delete index against alias - ignore unavailable":
+  - skip:
+      version: " - 5.99.0"
+      reason: delete index doesn't support aliases only from 6.0.0 on
+  - do:
+      indices.delete:
+        index: alias
+        ignore_unavailable: true
+  - do:
+      indices.get:
+        index: index
+  - do:
+      indices.get:
+        index: index,index2
+  - is_true: index
+  - is_true: index2
+---
+"Delete index against alias -  multiple indices":
+  - skip:
+      version: " - 5.99.0"
+      reason: delete index doesn't support aliases only from 6.0.0 on
+  - do:
+      catch: missing
+      indices.delete:
+        index: alias,index2
+  - do:
+      indices.get:
+        index: index,index2
+  - is_true: index
+  - is_true: index2
+---
+"Delete index against alias -  ignore unavailable - multiple indices":
+  - skip:
+      version: " - 5.99.0"
+      reason: delete index doesn't support aliases only from 6.0.0 on
+  - do:
+      indices.delete:
+        index: alias,index2
+        ignore_unavailable: true
+  - do:
+      indices.get:
+        index: index,index2
+        ignore_unavailable: true
+  - is_true: index
+  - is_false: index2
+---
+"Delete index against wildcard matching alias":
+  - skip:
+      version: " - 5.99.0"
+      reason: delete index doesn't support aliases only from 6.0.0 on
+  - do:
+      indices.delete:
+        index: alia*
+
+  - do:
```

Comment:
After double checking, these were leftover, they were all followed by another indices.get call, and for that one I was checking the response. Thanks for catching this.